### ISSUE TITLE
Provide a date and a time widget

### ISSUE THREAD

**Issue by kaikreuzer** [eclipse/smarthome#625]
migrated from Bugzilla [#423531](https://bugs.eclipse.org/bugs/show_bug.cgi?id=423531)
status NEW severity *enhancement* in component *UI* for *---*
Reported in version *unspecified* on platform *All*
Assigned to: Project Inbox

On 2013-12-08 09:44:58 -0500, Kai Kreuzer wrote nothing.
On 2013-12-08 09:45:13 -0500, Kai Kreuzer wrote:

> Migrated from https://code.google.com/p/openhab/issues/detail?id=490

**Comment by mkotsbak** [comment]
Yes please. Would be nice to set the leave time for an car engine heater to start some time before the set time.

**Comment by triller-telekom** [comment]
PaperUI already provides a time picker for the rule engine, see:

![image](https://user-images.githubusercontent.com/26876013/29923657-90dd7982-8e5a-11e7-8825-e87f5e3d1266.png)

**Comment by kaikreuzer** [comment]
This issue is about a "UI widget" for items, not about configurations (which didn't even exist at the time the issue was created...)

**Comment by triller-telekom** [comment]
Hmm in the original issue I read:

>Using hard-coded timestamps in rules is very inflexible. It should be possible to do certain actions until a specified date or at a specified time.

So I understand that the feature request was about being able to specify a time for a rule.

But I agree we do not have a Time/Datepicker in our basicui (yet).

**Comment by TimWeyand** [comment]
Added an 100$ Bounty (maximum 3 Month) for the basicui implementation

**Comment by kaikreuzer** [comment]
@lolodomo or @resetnow Any interest to work on this?

**Comment by asadath1395** [comment]
Is this still open?
